docs: add short description for adding a Sourcery badge to READMEs (#333)
sourcery-ai/sourcery: README.md

[![Sourcery](https://img.shields.io/badge/Sourcery-enabled-brightgreen)](https://sourcery.ai)

> Use the code below for copy badge image as follow:

Reviewer: I think the wording here can be improved a bit:

```suggestion
Copy the code below into your README to display your Sourcery badge:
```
